.github/workflows/static.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy static content to Pages
+
+on:
+  push:
+    branches: ['master']
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: 'pages'
+  cancel-in-progress: true
+
+jobs:
+  test:
+    uses: ./.github/workflows/run-tests.yml
+  build-and-deploy:
+    needs: test
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@8df78e55761745aad83acaf3ff12976382356e6d
+        with:
+          ruby-version: '.ruby-version'
+          bundler-cache: true
+      - name: Setup Node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'npm'
+      - name: Install and Build
+        run: |
+          npm ci
+          bundle exec middleman build
+      - name: Setup Pages
+        uses: actions/configure-pages@37538b280cbfab35da9dcd936a2b95f1f522d927
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@253fd476ed429e83b7aae64a92a75b4ceb1a17cf
+        with:
+          path: 'build'
```

Comment:
Just remembered - I think we were going to avoid using `upload-pages-artifact` given it doesn't really add anything useful? We can use the [Linux version of the Archive Artifact step](https://github.com/actions/upload-pages-artifact/blob/main/action.yml#L24-L33) and then use the generic [upload-artifact action](https://github.com/actions/upload-artifact) (that we already use in the [pay-smoke-tests repo](https://github.com/alphagov/pay-smoke-tests/blob/main/.github/workflows/post-merge.yml#L74), for example). This will ease the burden of managing the actions going forward, as we can review any Dependabot updates once across the repos and approve/merge in a batch.